rusk-wallet/src/cache.rs

```diff
@@ -155,6 +161,26 @@ impl Cache {
         }))
     }
 
+    pub(crate) fn insert_last_block_height(
+        &self,
+        block_height: u64,
+    ) -> Result<(), Error> {
+        let last_block_height = self.last_block_height()?.unwrap_or(0);
+        let max = std::cmp::max(last_block_height, block_height);
+
+        self.db.put(b"last_block_height", max.to_be_bytes())?;
```

Comment:
what happens if "last_block_height" was already stored? Will it update the previous value? Add a new one?

Reply:
Changed to `update_last_block_height`